### PR TITLE
Hotfix identity creator view bug

### PR DESCRIPTION
**Pull request by tddang-linagora**
## Before

https://github.com/user-attachments/assets/3cedf4d3-2a0d-4100-b050-9057c9f6785d

## After

https://github.com/user-attachments/assets/a54cc4c3-c1c1-4da7-9ddf-553cc8d5274f

### CODE REVIEW

**Comment by dab246** [comment]
- [ ] Please also check vacation

File: lib/features/identity_creator/presentation/identity_creator_view.dart [hunk 1]
```diff
@@ -472,10 +472,16 @@ class IdentityCreatorView extends GetWidget<IdentityCreatorController>
...
             ]
           ),
-          _buildHtmlEditorWeb(
-            context,
-            controller.contentHtmlEditor,
-            maxWidth),
+          ConstrainedBox(
+            constraints: BoxConstraints(
+              maxWidth: maxWidth,
+              maxHeight: 500,
```
Comment (dab246) [inline lib/features/identity_creator/presentation/identity_creator_view.dart:478]: Set default is `300px`

Comment (tddang-linagora) [reply lib/features/identity_creator/presentation/identity_creator_view.dart]: Done

**Comment by tddang-linagora** [comment]
>Please also check vacation

Vacation fixed

https://github.com/user-attachments/assets/ef7de989-a660-4d85-a875-760a6e71a423

**Comment by dab246** [comment]
## Root cause

- Side effect from https://github.com/linagora/tmail-flutter/pull/3280

**Comment by dab246** [comment]
- [ ] Please rebase it

**Review by hoangdat — APPROVED** [review]

**Pull request merged**